Dangerfile

```diff
@@ -10,15 +10,20 @@ no_changelog_entry = !git.modified_files.include?("CHANGELOG.md")
 no_test_changes = git.modified_files.grep(/Tests/).empty?
 
 if source_changes_exist && no_test_changes 
-    warn(“Consider adding tests for new extensions or updating existing tests for a modified SwifterSwift extension”)
+    warn("Consider adding tests for new extensions or updating existing tests for a modified SwifterSwift extension")
 end
 
 if source_changes_exist && no_changelog_entry
-    warn(“The source files have been modified. Please consider adding a CHANGELOG entry if necessary.“)
+    warn("The source files have been modified. Please consider adding a CHANGELOG entry if necessary.")
 end
 
+#Xcode summary
+xcode_summary.report 'xcodebuild-ios.json'
```

Comment:
We should also put these at the very end of the `Dangerfile`

Reply:
Sure :))